index.html

```diff
@@ -38,6 +39,204 @@ <h2 id="job-title">I am a Software Developer</h2>
                     <li><a href=""><img class="icon-interactions" src="images/Vector.png" alt="M-sign"></li></a>
                 </ul> 
             </section>
+            <section class="work">
+                <div>
+                    <div class="recent-work">
+                        <h2 id="work-intro">My Recent Work</h2>
+                        <a id="yoga-image" href=""><img src="images/Img Placeholder-1.png" alt="yoga girl"></a>
+                    </div>
+                    <div class="multi-post-div">
+                        <h2 id="multi-post-title" >Multi-Post Stories</h2>
+                        <div id="work-description">
+                            <p id="work-font">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        </div>
+                        <div class="button-div">
+                            <ul class="button-list">
+                                <li><a href=""><button id="multi-post-button">css</button></a></li>
+                                <li><a href=""><button id="multi-post-button">html</button></a></li>
+                                <li><a href=""><button id="multi-post-button">bootstrap</button></a></li>
+                                <li><a href=""><button id="multi-post-button">Ruby</button></a></li>
```

Comment:
Kindly, whenever you use a link label "to" put the correct link in the "href" attribute, if it does not have a real link, put a "#"

Reply:
Kindly, how much uses ID in any element, this must be unique on your page, if you need to apply any CSS style, it is more useful to use classes